tools/jit/templates/aten_dispatch.cpp

```diff
@@ -20,19 +20,53 @@ using operator_constructor = std::function<TensorOp(jit::Node*)>;
 
 namespace {
 
-void pack_list(std::vector<Tensor> & outputs, Tensor v) { outputs.push_back(v); }
-void pack_list(std::vector<Tensor> & outputs, Scalar v) { outputs.push_back(v.toTensor()); }
-void pack_list(std::vector<Tensor> & outputs, const std::vector<Tensor> & t) {
-  outputs.insert(outputs.end(), t.begin(), t.end());
+// a temporary Tensor that does not alter the refcount of impl on
+// acquisition or release, avoids any refcounting in dispatch functions
+struct TensorTemporary : public at::Tensor {
```

Comment:
Is this https://github.com/zdevito/ATen/issues/114 ?

Reply:
Yes, but I don't want to expose it in aten's header files because it will be very confusing to have two Tensor types, so it is just created locally in the dispatch implementation.